src/tests/entities/judging.test.ts

```diff
@@ -1,13 +1,12 @@
 import 'jest';
-import shuffle from 'shuffle-array';
-import { createDbConnection, closedbConnection } from './testdb';
-import { Team } from '../entities/team';
-import { Judge } from '../entities/judge';
-import { JudgingVote, insufficientVoteCountError } from '../entities/judgingVote';
-import logger from '../logger';
-import { createJudgeData, createTeamData } from './utilities';
+import { createDbConnection, closedbConnection } from '../testdb';
```

Comment:
I tried to ignore it, I swear, but can we capitalize the `d`?

```suggestion
import { createDbConnection, closeDbConnection } from '../testdb';
```

Reply:
Also just realized that it was like this before, meaning I approved this before

<img title="I am suddenly filled with regret" src="https://media.giphy.com/media/xVlewTCgGVs6xW16Dc/giphy.gif" width="200" alt="Soccer player holding his head, with the text what have I done" />

Reply:
Yea good catch, I’ll fix that